src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -1,42 +1,50 @@
 load("//:build/wd_test.bzl", "wd_test")
 
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+copy_file(
+    name = "pyodide-2.capnp.bin@rule",
+    src = "//src/pyodide:pyodide.capnp.bin",
```

Comment:
I think this should be version `"dev"` or something like that, since it uses the bundle from workerd source. To get version 2 we should download it with bazel. If we're not ready to do that, maybe just call this `pyodide-dev.capnp.bin` and set the version to `"dev"` in the code. Don't want to confuse ourselves more than necessary.

Reply:
A comment in the bazel file here to explain this also wouldn't go amiss :)